tests/integration/docusaurus/expectations/advanced/multi_batch_rule_based_profiler_example.py

```diff
@@ -37,8 +37,8 @@
       - expectation_type: expect_table_row_count_to_be_between
         class_name: DefaultExpectationConfigurationBuilder
         module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
-        min_value: $parameter.row_count_range.value.min_value
```

Comment:
Just confirming - this was already broken, but was just getting skipped?

Reply:
Yes it was broken, but wasn't getting detected.